arch/arm/Kconfig

```diff
@@ -593,6 +576,23 @@ config ARCH_CHIP_STR71X
 	---help---
 		STMicro STR71x architectures (ARM7TDMI).
 
+config ARCH_CHIP_CSK6
```

Comment:
Shouldn't this be alphabetically sorted?

Reply:
done